echopype/tests/mask/test_mask.py

```diff
@@ -1240,4 +1241,146 @@ def test_apply_mask_channel_variation(source_has_ch, mask, truth_da):
+
+    # Compute MVBS
+    MVBS = ep.commongrid.compute_MVBS(
+        ds_Sv=ds_Sv,
+        range_var=range_var,
+        range_bin="1m",
+        ping_time_bin="10s"
+    )
+
+    if use_multi_channel_mask:
+        # Create two frequency differencing masks that in total cover channels
+        # 18kHz - 200kHz:
+        # (120kHz - 38kHz) and (38kHz - 18kHz)
+        diff = [
+            ep.mask.frequency_differencing(
+                source_Sv=MVBS,
+                freqABEq="120000.0Hz - 38000.0Hz > 0dB",
+                storage_options={},
+            ).expand_dims(dim={"channel": MVBS["channel"][0:2].data}),
+            ep.mask.frequency_differencing(
+                source_Sv=MVBS,
+                freqABEq="38000.0Hz - 18000.0Hz > 0dB",
+                storage_options={},
+            ).expand_dims(dim={"channel": MVBS["channel"][2:6].data}),
+        ]
+
+        # Apply frequency differencing mask to MVBS
+        MVBS_freq_diff = ep.mask.apply_mask(MVBS, diff)
+
+        # Check masked MVBS values
+        assert np.allclose(
+            xr.where(diff[0], MVBS["Sv"].isel(channel=slice(0,2)), np.nan),
+            MVBS_freq_diff["Sv"].isel(channel=slice(0,2)),
+            equal_nan=True
+        )
+        assert np.allclose(
+            xr.where(diff[1], MVBS["Sv"].isel(channel=slice(2,6)), np.nan),
+            MVBS_freq_diff["Sv"].isel(channel=slice(2,6)),
+            equal_nan=True
+        )
+    else:
+        # Create frequency differencing (120kHz - 38kHz) mask
+        diff = ep.mask.frequency_differencing(
+            source_Sv=MVBS,
+            freqABEq="120000.0Hz - 38000.0Hz > 0dB",
+            storage_options={},
+        )
+
+        # Apply frequency differencing mask to MVBS
+        MVBS_freq_diff = ep.mask.apply_mask(MVBS, diff)
+
+        # Check masked MVBS values
+        assert np.allclose(
+            xr.where(
+                diff.expand_dims(dim={"channel": MVBS["channel"].data}),
+                MVBS["Sv"],
+                np.nan
+            ),
+            MVBS_freq_diff["Sv"],
+            equal_nan=True
+        )
+
+    # Check dimensions
+    assert MVBS_freq_diff["Sv"].dims == ("channel", "ping_time", range_var)
+
+
+@pytest.mark.unit
+def test_validate_source_ds_and_check_mask_dim_alignment():
+    """
+    Tests that ValueErrors are raised for `_validate_source_ds_and_check_mask_dim_alignment`.
+    """
+    # Parse Raw File
+    ed = ep.open_raw(
+        raw_file="echopype/test_data/ek60/DY1801_EK60-D20180211-T164025.raw",
+        sonar_model="EK60"
+    )
+
+    # Compute Sv
+    ds_Sv = ep.calibrate.compute_Sv(ed)
+
+    # Compute MVBS
+    MVBS = ep.commongrid.compute_MVBS(
+        ds_Sv=ds_Sv,
+        range_bin="1m",
+        ping_time_bin="10s"
+    )
+
+    # Create frequency differencing (120kHz - 38kHz) mask
+    diff = ep.mask.frequency_differencing(
+        source_Sv=MVBS,
+        freqABEq="120000.0Hz - 38000.0Hz > 0dB",
+        storage_options={},
+    )
```

Comment:
Same as the above -- seems better to take out the dependency on `frequency_differencing`.